code/FulpstationCode/dice_fulp.dm

```diff
@@ -17,10 +17,10 @@
 	var/owner = null
 	dice_spawn = /mob/living/simple_animal/hostile/carp
 
-obj/item/dice/encounter/attack_self(mob/user)
+/obj/item/dice/encounter/attack_self(mob/user)
 	diceroll(user)
 
-obj/item/dice/encounter/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+/obj/item/dice/encounter/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
```

Comment:
if you don't have these the linter thinks you are goon code, and shoots you for existing.

Reply:
as it should